README.md

```diff
@@ -28,6 +28,7 @@ Property                           | Description
 `jasmineExplorer.config`           | The location of the Jasmine config file (relative to the workspace folder) (default: `spec/support/jasmine.json`)
 `jasmineExplorer.env`              | Environment variables to be set when running the tests
 `jasmineExplorer.nodePath`         | The path to the node executable to use. By default it will attempt to find it on your PATH, if it can't find it or if this option is set to `null`, it will use the one shipped with VS Code
+`jasmineExplorer.nodArgv`          | The arguments to the node executable to use.  To enable automatic typescript, import ts-node package and use these arguments: [\"--require\", \"ts-node/register\"]
```

Comment:
Looks like a typo here -- should be `jasmineExplorer.nodeArgv`, not `jasmineExplorer.nodArgv`

Reply:
Nice catch.